### PR TITLE
fixes #2894

### LINKED ISSUE #2894: small UX problem when choosing a ticket validator
Hi

I notice the UX is not the best when choosing a validator for a ticket

image

The 3rd field where one chooses a user should be wider bu default. Even the account glpi is too long.

### PR DESCRIPTION
**Pull request by rcrdmedina**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #2894

*Please update this template with something that matches your PR*

### CODE REVIEW

**Comment by rcrdmedina** [comment]
I am a newbie working with forked repository, I had deleted my forked repository, I made a new pr

**Comment by rcrdmedina** [comment]
excuse me

**Comment by trasher** [comment]
OK, no problem ;)

File: css/jquery-glpi.css [hunk 1]
```diff
@@ -174,7 +174,9 @@ html .ui-autocomplete {
 .select2-container .select2-choice div b {
     margin-top: -2px;
 }
-
+#show_validator_field .select2-container{
+  width: 60% !important;
```
Comment (trasher) [inline css/jquery-glpi.css]: Why `!important`?

**Comment by trasher** [comment]
Manually merged

**Pull request closed without merging**